src/ZendeskApi.Client/Resources/UserIdentityResource.cs

```diff
@@ -28,9 +29,7 @@ public IResponse<UserIdentity> Post(UserIdentityRequest request)
 
         public IResponse<UserIdentity> Put(UserIdentityRequest request)
         {
-            var response = Post(request);
```

Comment:
HI,

I think this PR is fine. Put and Post will work as expected - using the correct HTTP verbs. Users of this client who want to update a zendesk user's email address will have to post that new user, delete the old, then reverify. This is what the zendesk api documentation tells you you will have to do.

I don't think you're code above will work anymore if you get the 1.0.* version of the nuget package as the put won't be able to update the user's email. You'll have to do a post, then a delete in your 'else' statement. Then you'll have to set the 'verified' value accordingly.